superchain/superchain.go

```diff
@@ -18,6 +21,12 @@ var superchainFS embed.FS
 //go:embed extra/addresses extra/bytecodes extra/genesis extra/genesis-system-configs
 var extraFS embed.FS
 
+//go:embed implementations/networks implementations
+var implementationsFS embed.FS
```

Comment:
Also, go embeds files in directories [recursively](https://pkg.go.dev/embed), so this can just be `implementations`.
Though, note it will exclude dotfiles and `_` prefixed files. If we want to include this, we can use the `all:` directive prefix.
```suggestion
//go:embed implementations
var implementationsFS embed.FS
```